src/main/java/bio/terra/service/job/StairwayExceptionFieldsFactory.java

```diff
@@ -47,8 +52,8 @@ private static String getStepNameFromStairwayException(Exception ex) {
           break;
         }
       } catch (ClassNotFoundException e) {
-        throw new ExceptionSerializerException(
-            "All steps in a stacktrace should be able to be found", e);
+        // Unknown classes are possible in a stack trace: log, but keep walking the path.
+        logger.warn("Could not resolve StackTraceElement to a class", e);
```

Comment:
I was on the fence about the log level: I am open to downgrading it to `info` or `debug` but I don't think this will happen all that often.

Reply:
I agree that it makes sense as a warning for now